build.gradle.kts

```diff
@@ -125,9 +115,10 @@ subprojects {
         }
     }
 
-    tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
-        kotlinOptions {
-            freeCompilerArgs = listOf("-Xopt-in=kotlin.time.ExperimentalTime")
+    configure<KotlinJvmProjectExtension> {
+        jvmToolchain(17)
```

Comment:
I found that the project was building with java 19 support locally and this corrected the jvm setting in the subprojects.